pkg/operator/controllers/checker/machinechecker.go

```diff
@@ -75,7 +75,7 @@ func (r *MachineChecker) machineValid(ctx context.Context, machine *machinev1bet
 		return []error{fmt.Errorf("machine %s: failed to read provider spec: %T", machine.Name, o)}
 	}
 
-	if !validate.VMSizeIsValid(api.VMSize(machineProviderSpec.VMSize), r.isLocalDevelopmentMode, isMaster) {
+	if !v20210131preview.VMSizeIsValid(v20210131preview.VMSize(machineProviderSpec.VMSize), r.isLocalDevelopmentMode, isMaster) {
```

Comment:
To allow the machine checker to be able to validate the new types on the aro operator, the new VM types must be included in the `VMSizeIsValid` function that now lives with the preview api with the new addition of the vm types.